app/web/components/Navigation/ReportButton.tsx

```diff
@@ -169,9 +170,11 @@ export default function ReportButton({
                 {t("report.bug.button_label")}
               </StyledReportTypeButton>
               <StyledReportTypeButton
-                onClick={() => {
-                  setType("content");
-                }}
```

Comment:
Looks like this was the only place we're setting the `content` type, so we can do a little clean up here too:

- Remove `content` from the type for the useState at the top of the component
- Remove the `type === content` conditional rendering logic